azurerm/resource_arm_application_gateway.go

```diff
@@ -547,7 +547,23 @@ func resourceArmApplicationGateway() *schema.Resource {
 					},
 				},
 			},
-
+			"autoscale_configuration": {
+				Type:     schema.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"min_capacity": {
```

Comment:
We should add some validation for greater than zero and some max value. I can't find that information in the portal or the docs but testing for it should be fairly straightforward by just passing in a ridiculously large number. The API normally throws an error and lets the user know what the max value for that attribute is.

Reply:
Sure, can do. I didn't add it as I couldn't find a limit in docs the same as you, but will experiment when I can get back to this 👍

Reply:
The portal allows between 2 and 10 for minimum, I've pushed a commit for this

Reply:
Great! That looks good